piccolo/columns/column_types.py

```diff
@@ -315,7 +315,7 @@ class Band(Table):
 
     def __init__(
         self,
-        length: int = 255,
+        length: int | None = 255,
```

Comment:
Can we make this `t.Optional[int]` instead? Just because we still support Python 3.9, and the `|` operator isn't available in that version.